### ISSUE TITLE
Integrate the highlight gcode feature in the official gedit source code!

### ISSUE THREAD

**Issue by Obijuan** [jvegaf/gcode-gedit#6]

**Comment by Obijuan** [comment]
It would be great if gedit includes this feature by default, without having to install anything  :-)

**Comment by jvegaf** [comment]
yes, would be great, or another idea,  like a plugin, through a .deb package and include on a repository, then you only have put $apt-get install gcode-gedit on your favourite terminal or include on another installer script
